### PR TITLE
BETA: Register nah.is-a.dev

### PR DESCRIPTION
**Pull request by HoaQuyingVN**
Added `nah.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by HoaQuyingVN** [comment]
Wait

**Review by HoaQuyingVN — COMMENTED** [review]
I changed

File: domains/share.nazi.json
Comment (DEV-DIBSTER) [inline domains/share.nazi.json:1]: Yeah.... absolutely not.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Comment by DEV-DIBSTER** [comment]
@phenax Requesting a ban from the repository due to anti-semitic domain name request.

File: domains/slag.json
Comment (WilliamDavidHarrison) [inline domains/slag.json:1]: This is a no as well.

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by phenax** [comment]
User has been banned

**Pull request closed without merging**